Kiosk/Bid Fulfillment/ConfirmYourBidPINViewController.swift

```diff
@@ -65,15 +66,20 @@ class ConfirmYourBidPINViewController: UIViewController {
                         .updateUserCredentials(loggedInProvider)
                         .mapReplace(provider)
                 }
-                .flatMap { provider -> Observable<AuthorizedNetworking> in
+                .flatMap { provider -> Observable<BypassResults> in
```

Comment:
I've been looking for an alternative to how we do this now with `.mapReplace(provider)`, and this isn't it. I'd rather keep this consistent with what we do now.